clap_builder/src/builder/command.rs

```diff
@@ -1792,7 +1792,8 @@ impl Command {
     /// This is often used to describe how to use the arguments, caveats to be noted, or license
     /// and contact information.
     ///
-    /// If [`Command::after_help`] is not specified, this message will be displayed for `-h`.
+    /// This message is not displayed for `-h`. Use [`Command::after_help`] to show message
```

Comment:
Could you also update `before_long_help`?

Reply:
I'll make sure all methods listed in the table above are in sync.